tests/integration-service/namespace-backed-environments.go

```diff
@@ -59,6 +72,15 @@ var _ = framework.IntegrationServiceSuiteDescribe("Namespace-backed Environment
 
 			integrationTestScenario, err = f.AsKubeAdmin.IntegrationController.CreateIntegrationTestScenarioWithEnvironment(applicationName, testNamespace, gitURLForNBE, revisionForNBE, pathInRepoForNBE, userPickedEnvironment)
 			Expect(err).ShouldNot(HaveOccurred())
+			phaseDTC = "Bound"
+			phaseDT = "Bound"
```

Comment:
I think we should use defined constants instead of hardcoded values
```suggestion
			phaseDTC = appstudioApi.DeploymentTargetClaimPhase_Bound
			phaseDT = appstudioApi.DeploymentTargetPhase_Bound
```